rest-api/offline/biobank_samples_pipeline.py

```diff
@@ -306,6 +306,18 @@ def _get_hpo_type_sql(hpo_alias):
   result += "ELSE 'UNKNOWN' END)"
   return result
 
+_NATIVE_AMERICAN_SQL = """
+  (SELECT (CASE WHEN count(*) > 0 THEN 'Y' ELSE 'N' END)
+       FROM questionnaire_response qr
+       INNER JOIN questionnaire_response_answer qra
+         ON qra.questionnaire_response_id = qr.questionnaire_response_id
+       INNER JOIN questionnaire_question qq
+         ON qra.question_id = qq.questionnaire_question_id
+      WHERE qr.participant_id = participant.participant_id
```

Comment:
Where did "participant" come from here - is this assuming something in the outer scope? If so, should this string be templated instead to make the dependency clear?

Reply:
It does assume that you have "participant" as a table name in a join elsewhere. I'm not sure that using a template would make this clearer, though. I'll add a comment.